src/Entity/LineItem.php

```diff
@@ -64,12 +63,12 @@ public function __construct(
         $this->taxRate = $taxRate;
     }
 
-    public function getId()
+    public function getId(): ?int
```

Comment:
I would be for leaving it as it was before, without return type to keep id more customizable and consistent with all Sylius resources. I know that our mapping is configured for using integers, but it is relatively easy to change the strategy to use for example UUIDs instead of integers.

Reply:
I would agree that `getId(): ?int` is opinionated, but not because somebody might want to make that UUID, but because sylius resource bundle doesn't define type-hint.

Though this is concrete implementation that you're marking *final*. In such case it's totally alright to make it `int` and not to support some doctrine hacks for 3rd party.
![image](https://user-images.githubusercontent.com/870747/144574645-256f7ec9-51d3-425d-bd82-86ec338d1f03.png)

Reply:
Did you see, that BillingData id is already `int` in class, but `ShopBillingData` is not? This is inconsistency and my PR is solving this as well.

Reply:
Should `Invoice`'s `protected string $id` became `@mixed` as well and constructor's `string` type-hint removed, because somebody might want to change DB id to int?